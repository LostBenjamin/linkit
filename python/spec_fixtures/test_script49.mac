$value 3

[macro 10]
  blue
  ((test <value> `<value>*3`))

[[test X Y
  [macro-X]
    `Y*2`
     add-palette-color
     refresh-display
]]

