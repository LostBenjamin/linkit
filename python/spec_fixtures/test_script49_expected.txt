10:set:blu
4:set:18:pal:flu

